modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -0,0 +1,30 @@
+import Base from '../../../../../../assets/dev/js/frontend/handlers/base';
```

Comment:
```suggestion
import Base from 'elementor/assets/dev/js/frontend/handlers/base';
```
@davseve Does this work?